extensions/sql-database-projects/src/controllers/mainController.ts

```diff
@@ -56,6 +57,10 @@ export default class MainController implements vscode.Disposable {
 		vscode.commands.registerCommand('sqlDatabaseProjects.build', async (node: WorkspaceTreeItem) => { return this.projectsController.buildProject(node); });
 		vscode.commands.registerCommand('sqlDatabaseProjects.publish', async (node: WorkspaceTreeItem) => { return this.projectsController.publishProject(node); });
 		vscode.commands.registerCommand('sqlDatabaseProjects.schemaCompare', async (node: WorkspaceTreeItem) => { return this.projectsController.schemaCompare(node); });
+		vscode.commands.registerCommand('sqlDatabaseProjects.schemaCompareGetTargetScripts', async (projectFilePath: string): Promise<string[]> => { return await this.projectsController.schemaCompareGetTargetScripts(projectFilePath); });
+		vscode.commands.registerCommand('sqlDatabaseProjects.getProjectDatabaseSchemaProvider', async (projectFilePath: string): Promise<string> => { return await this.projectsController.getProjectDatabaseSchemaProvider(projectFilePath); });
```

Comment:
I think it would make more sense to expose these in projectProvider.ts rather than have them as commands since they're being used directly by another extension, not used through command palette or a right-click menu by the user.

Reply:
I think you're right.  Think that's something that could be done after this gets checked in so that it's guaranteed to meet askmode?  I think it'd be okay if that changeover ends up happening for Feb due to holiday rush and my OoF.

Reply:
I'd prefer it get done right before checking in, but sure it's ok since we're tight on time (as long as this actually gets fixed soon).

Reply:
Aight, I've swapped the commands for ProjectProvider functions.